wrappers/python/pysetup.cmake.in

```diff
@@ -1,8 +1,13 @@
 set(ENV{OPENMM_INCLUDE_PATH} "@WRAPPER_INCLUDE_DIRS@")
+set(ENV{CC} "@CMAKE_C_COMPILER@")
+set(ENV{CXX} "@CMAKE_CXX_COMPILER@")
 file(TO_NATIVE_PATH "@CMAKE_INSTALL_PREFIX@/lib" OPENMM_LIB_PATH)
 set(ENV{OPENMM_LIB_PATH} "${OPENMM_LIB_PATH}")
 message("OPENMM_LIB_PATH = " $ENV{OPENMM_LIB_PATH})
 message("OPENMM_INCLUDE_PATH = " $ENV{OPENMM_INCLUDE_PATH})
+message("CC = " $ENV{CC})
+message("CXX = " $ENV{CXX})
```

Comment:
It works just fine to build python extensions with gcc/icc/clang/whatever, since it's C and it's all ABI compatible.